developer_manual/app/fundamentals/database.rst

```diff
@@ -236,15 +236,14 @@ Database Migrations
 
 .. sectionauthor:: Thomas Müller <thomas.mueller@tmit.eu>
 
-ownCloud uses migration steps to perform any changes between versions. 
-In most cases, these changes are changes to the database schema, but it is not only limited to them.
-Therefore we support three kinds of migration steps: simple, SQL and schema.
+ownCloud uses migration steps to perform database changes between releases. 
```

Comment:
@settermjd as for **Simple** migration it indeed can be unrelated to database, so it's better do not replace `any` with `database` here.
cc @DeepDiver1975

Reply:
Ok, I'll change it back — but since the information's in a section titled "Database Migrations", the lack of specificity may cause confusion, no?